src/rules/no-unused-modules.js

```diff
@@ -51,21 +50,81 @@ function requireFileEnumerator() {
 }
 
 /**
- *
+ * Given a FileEnumerator class, instantiate and load the list of files.
  * @param FileEnumerator the `FileEnumerator` class from `eslint`'s internal api
  * @param {string} src path to the src root
  * @param {string[]} extensions list of supported extensions
  * @returns {{ filename: string, ignored: boolean }[]} list of files to operate on
  */
 function listFilesUsingFileEnumerator(FileEnumerator, src, extensions) {
-  const e = new FileEnumerator({
-    extensions,
-  });
+  // We need to know whether this is being run with flat config in order to
+  // determine whether we need to customize the FileEnumerator or not.
+
+  // This condition is sufficient to test in v8, since the environment variable
+  // is necessary to turn on flat config
+  let isUsingFlatConfig =
+    process.env.ESLINT_USE_FLAT_CONFIG &&
+    process.env.ESLINT_USE_FLAT_CONFIG !== 'false';
```

Comment:
```suggestion
  let isUsingFlatConfig = process.env.ESLINT_USE_FLAT_CONFIG &&
    process.env.ESLINT_USE_FLAT_CONFIG !== 'false';
```
linebreaks after the `=` is a signature prettier thing :-(

Reply:
Updated